src/main/kotlin/CheckersBoard.kt

```diff
@@ -0,0 +1,65 @@
+class CheckersBoard(private val boardSize: Int) : BaseBoard {
+    private val board = Array(boardSize * boardSize) {i -> Square(if ((i / boardSize + i % boardSize) % 2 == 0) -1 else 1)}
+    init {
+        setStartPosition()
+    }
+
+    private fun getIndex(i : Int, j : Int): Int {
+        return i * boardSize + j;
+    }
+
+    override fun setStartPosition() {
+        for (i in 0 until boardSize) {
+            for (j in 0 until boardSize) {
+                if (i < 3 && board[getIndex(i, j)].color == -1) {
+                    board[getIndex(i, j)].figure = Figure(1)
```

Comment:
You may write `get(i, j)` and `set(i, j)` instead. Another option is to write `this[i,j]`. I think, it will look better

Reply:
Fixed